examples/logging/advanced_requests.md

```diff
@@ -0,0 +1,76 @@
+By default django logs [4xx and 5xx requests](https://docs.djangoproject.com/en/3.2/topics/logging/#django-request) 
+but is missing details such as the request user name.  This can be helpful in debugging. This example 
+shows how the request user can be added to the logs.
+
+```python
+def add_username(record):
+    record.username = record.request.user.username
+    return True
+
+
+LOG_LEVEL = "DEBUG" if DEBUG else "INFO"
+LOGGING = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "formatters": {
+        "normal": {
+            "format": "%(asctime)s.%(msecs)03d %(levelname)-7s %(name)s: %(message)s",
+            "datefmt": "%H:%M:%S",
+        },
+        "request": {
+            "format": "%(asctime)s.%(msecs)03d %(levelname)-7s %(name)s: user=%(username)s %(message)s",
+            "datefmt": "%H:%M:%S",
+        },
+    },
+    "handlers": {
+        "normal_console": {
+            "level": "DEBUG",
+            "class": "logging.StreamHandler",
+            "formatter": "normal",
+        },
+        "requests": {
+            "level": "DEBUG",
+            "class": "logging.StreamHandler",
+            "formatter": "request",
+            "filters": ["add_username"],
+        },
+    },
+    "filters": {
+        "add_username": {
+            "()": "django.utils.log.CallbackFilter",
+            "callback": add_username,
+        }
+    },
+    "loggers": {
+        "django.request": {
+            "handlers": ["requests"],
+            "level": LOG_LEVEL,
+            "propagate": False,
+        },
+        "django": {"handlers": ["normal_console"], "level": "INFO"},
+        "nautobot": {
+            "handlers": ["normal_console"],
+            "level": LOG_LEVEL,
+        },
+    },
+}
+```
+
+## Example Logs
+
+```no-highlight
+20:50:57.972 INFO    django.server:  "GET /health/ HTTP/1.1" 200 11743
+20:50:57.972 INFO    django.server:  "GET /health/ HTTP/1.1" 200 11743
```

Comment:
Why are the logs duplicated in this example? Is that something that could be fixed?